lexicons/bsky.app/updateProfile.json

```diff
@@ -0,0 +1,37 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.updateProfile",
+  "type": "procedure",
+  "description": "Notify server that the user has seen notifications",
+  "parameters": { },
+  "input": {
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": [],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "maxLength": 64
+        },
+        "description": {
+          "type": "string",
+          "maxLength": 256
+        },
+        "badges": {"type": "array", "items": {"$ref": "lex:app.bsky.profile#/defs/badgeRef"}}
+      }
+    }
+  },
+  "output": { 
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": ["uri", "cid", "record"],
+      "properties": {
+        "uri": {"type": "string"},
+        "cid": {"type": "string"},
+        "record": {"type": "object"}
```

Comment:
Is the idea that this matches the generic CRUD interface for updates?

Reply:
it's only the `Update` method & it takes a partial Profile.
CRD can still be done the normal way. this is jsut the one record that we currently allow edits of